browser/themes/BUILD.gn

```diff
@@ -1,26 +1,40 @@
 source_set("themes") {
-  sources = [
-    "brave_dark_mode_utils.h",
-    "brave_dark_mode_utils_android.cc",
-  ]
+  sources = [ "brave_dark_mode_utils.h" ]
 
   deps = [
     "//base",
     "//brave/common:pref_names",
     "//brave/common:switches",
   ]
 
+  if (is_android) {
+    sources += [ "brave_dark_mode_utils_android.cc" ]
+  }
+
```

Comment:
if/else

Reply:
Fixed in f02321e2e26825fe16c1737b259f71ff35c5aea3